src/rdkafka_event.c

```diff
@@ -250,6 +247,9 @@ int rd_kafka_event_debug_contexts (rd_kafka_event_t *rkev,
                 NULL
         };
 
+        if (unlikely(rkev->rko_evtype != RD_KAFKA_EVENT_LOG))
+                return -1;
+
         int ctx = rkev->rko_u.log.ctx;
```

Comment:
This variable def needs to go above the if- code too.

Older MSVC compilers will not accept inline variable definitions.

Reply:
There is no need for this variable, btw, just reference the union directly below.

Reply:
ok, thats likely why the win32 test was failing